src/main/scala/cognite/spark/v1/AssetsRelation.scala

```diff
@@ -80,24 +80,12 @@ class AssetsRelation(config: RelationConfig)(val sqlContext: SQLContext)
     }
   override def getFromRowAndCreate(rows: Seq[Row]): IO[Unit] = {
     val assets = fromRowWithFilteredMetadata(rows)
-
-    client.assets
-      .create(assets)
-      .handleErrorWith {
-        case e: CdpApiException =>
-          if (e.code == 409) {
-            val existingExternalIds =
-              e.duplicated.get.map(j => j("externalId").get.asString.get)
-            resolveConflict(existingExternalIds, assets)
-          } else { IO.raiseError(e) }
-      } *> IO.unit
-  }
-
-  def resolveConflict(existingExternalIds: Seq[String], assets: Seq[AssetCreate]): IO[Unit] =
     upsertAfterConflict[Asset, AssetUpdate, AssetCreate, Assets[IO]](
-      existingExternalIds,
+      Seq.empty,
       assets,
-      client.assets)
+      client.assets,
+      doUpsert = true)
```

Comment:
Perhaps we should give a default value to this parameter, since it's only set to `false` when handling duplicates?